content/release-10-15-0/devices-10-15-0-bundle/10-15-0-377.md

```diff
@@ -0,0 +1,37 @@
+---
+weight: 19
+title: Release 10.15.0.377
+layout: bundle
+---
+
+<!--10.15.0.369 - 10.15.0.377-->
+
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>LWM2M</td>
+<td>Custom actions were not always executed if a LWM2M device used a SEND operation to report the whole LWM2M object data. This issue is now resolved and the custom actions are always executed for the resource instances for which they are defined.</td>
+<td>DM-2188</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M agent is now able to properly convert the timestamps from the SenML data reported by the LWM2M client to a platform compatible date-time format for performing respective resource actions.</td>
```

Comment:
date-time vs datetime (not sure yet)

also, "platform" probably refers to Cumulocity IoT platform, it might be useful to add it here

Reply:
Seems sufficient to me in the context of the Cumulocity IoT release notes